src/com/owncloud/android/files/FileOperationsHelper.java

```diff
@@ -205,10 +207,14 @@ public void sendDownloadedFile(OCFile file) {
         }
     }
 
+    public void syncFiles(ArrayList<OCFile> files) {
```

Comment:
If we want to be completely implementation independent then this should be Collection\<OCFile\> and the same apply to toggleFavorites